index.md

```diff
@@ -22,6 +22,10 @@ scala3-sections:
         description: "Learn Scala by reading a series of short lessons."
         icon: "fa fa-book-open"
         link: /scala3/book/introduction.html
+      - title: "Scala for Java Programmers"
```

Comment:
we already have an article with this title at https://docs.scala-lang.org/tutorials/scala-for-java-programmers.html, this could be confusing